docs/thrust/api_docs/utility/type_traits.rst

```diff
@@ -7,7 +7,6 @@ Type Traits
    :glob:
    :maxdepth: 1
 
-   ${repo_docs_api_path}/*struct*integer__sequence*
```

Comment:
A bit of grepping the results from build the docs makes me believe that `integer_sequence` is now part of the `typedef_group__type__traits_....rebreather_rst` files and should be covered by the selector `${repo_docs_api_path}/*typedef_group__type__traits*` below.